src/test/java/org/candlepin/subscriptions/tally/MetricUsageCollectorTest.java

```diff
@@ -30,12 +30,7 @@
 import static org.mockito.Mockito.when;
 
 import java.time.OffsetDateTime;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Please omit this from the commit since the rest of the class doesn't actually have any changes

Reply:
Will Do